.github/workflows/full-ci-dummy.yml

```diff
@@ -0,0 +1,25 @@
+name: "Full CI"
+on:
+  pull_request:
+    branches:
+      - '*'  # must quote since "*" is a YAML reserved character; we want a string
+
+    ##############################################################################
+    #                                                                            #
+    #   This section must be synchronized with 'paths-ignore' in full-ci.yml     #
```

Comment:
maybe mod comment here and in other to save at least myself not having to think quite as hard about what has to be compared...
"This 'paths' section .... with (all | both) 'paths-ignore' sections in full-ci.yml."